pylmod/base.py

```diff
@@ -13,15 +13,15 @@
 
 class Base(object):
     """
-    The Base class provides the transport for accessing MIT LM web service.
+    The Base class provides the transport for accessing MIT LMod web service.
```

Comment:
Maybe worth doing the `Learning Modules (LMod)` and then using (LMod) in the rest of the class convention.